packages/website/src/components/DocsInnerNavigation.tsx

```diff
@@ -1,4 +1,5 @@
 // TODO: Refactor this mess
+/* eslint-disable react-hooks/rules-of-hooks, react-hooks/exhaustive-deps */
```

Comment:
I've ignored the rules here and also [here](https://github.com/reakit/reakit/pull/485/commits/69cab0cb1fddcd5f2db01c1f21f73c829afddd5f#diff-9ab67654aaf3aec927aeffee471b5fb2R2) because it needs to be refactored anyway :laughing: